### ISSUE TITLE
Training code for Oxford dataset

### ISSUE THREAD

**Issue by lbhwyy** [zjuluolun/BEVPlace#15]
Great work, and thank you for open-sourcing this project! I was able to reproduce the results on the KITTI dataset, but I encountered some issues with the Oxford dataset.

When I tried training Bevplace++ on the Oxford dataset, I only achieved a Recall@1 result of 73. I used the default code for generating BEV images (with path adjustments to fit the Oxford dataset), and the learning rate and batch size were also set to the default values. I believe something might be wrong. Could you please consider open-sourcing the training code for the Oxford dataset?

**Comment by zjuluolun** [comment]
Hi @lbhwyy, thank you for your interest! The point clouds in the Oxford dataset differ from those in KITTI and NCLT because their coordinates are normalized to the range [-1, 1]. To generate accurate BEV images, you'll need to denormalize the points. In our implementation, we scale the point values by a factor of 40, which results in BEV images of size 200x200. I suggest comparing the BEV images from the Oxford and KITTI datasets to ensure the format is correct.

If you continue to encounter issues with reproducing the results, feel free to add me on WeChat (llluolun) for further discussion.

**Comment by lbhwyy** [comment]
Thank you for your kindly reply!
